common/document_parser/lib/section_parse/parsers/dod_parser.py

```diff
@@ -0,0 +1,609 @@
+        return ["\n".join(s) for s in self._sections]
+
+    @property
+    def purpose(self):
+        return self._get_section_by_title("purpose")
+
+    @property
+    def responsibilities(self):
+        return self._get_section_by_title("responsibilities")
+
+    @property
+    def subject(self):
+        return self._get_section_by_title("subject")
+
+    @property
+    def references(self):
+        return self._get_section_by_title("references")
+
+    @property
+    def procedures(self):
+        return self._get_section_by_title("procedures")
+
+    @property
+    def effective_date(self):
+        return self._get_section_by_title("effective date")
+
+    @property
+    def applicability(self):
+        applicability_section = [
+            section.split("\n")
+            for section in self._get_section_by_title("applicability")
+        ]
+
+        # Sometimes the Applicability section is a subsection of Section 1.
+        if not applicability_section:
+            section_1 = self._get_section_by_num(1)
+            applicability_section = get_subsection_of_section_1(
+                section_1, "applicability"
+            )
+
+        return applicability_section
+
+    @property
+    def policy(self):
+        policy_section = self._get_section_by_title("policy")
+
+        # Sometimes the Policy section is a subsection of Section 1.
+        if not policy_section:
+            section_1 = self._get_section_by_num(1)
+            policy_section = get_subsection_of_section_1(section_1, "policy")
+
+        return policy_section
+
+    @property
+    def organizations(self):
+        return self._get_section_by_title("organizations")
+
+    @property
+    def definitions(self):
+        return self._get_section_by_title("definitions")
+
+    @property
+    def table_of_contents(self):
+        return self._get_section_by_title("table of contents")
+
+    @property
+    def authorities(self):
+        return self._get_section_by_title("authorities")
+
+    @property
+    def glossary(self):
+        return self._get_section_by_title("glossary")
+
+    @property
+    def releasability(self):
+        return self._get_section_by_title("releasability")
+
+    @property
+    def summary_of_change(self):
+        return self._get_section_by_title("summary of change")
+
+    def _parse(self) -> None:
+        raw_text = self._get_raw_text()
+        if not raw_text:
+            return
+
+        self._sections = [[line] for line in raw_text.split("\n")]
+        self._combine_toc()
+        self._remove_pagebreaks_and_noise()
+        self._combine_enclosure_titles()
+        self._combine_sentence_continuations()
+        self._combine_alpha_list_items()
+        self._combine_reference_list()
+        self._combine_by_section_nums()
+        self._combine_enclosures()
+        self._combine_glossary_then_references()
+        self._remove_repeated_section_titles()
+        self._combine_enclosures_list()
```

Comment:
Hey Olive: For these, can you add a comment above each just mentioning what each function is doing and how it is changing the `_sections` items for the document?

Reply:
I just pushed some docstrings/ comments in 926a5624bd53400ebb4391fd18bcf84abe70d5ec. Is that sufficient, or do you think I should add comments here too?